anclaHelper.go

```diff
@@ -0,0 +1,44 @@
+package ancla
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	// nolint:staticcheck
+	"github.com/xmidt-org/webpa-common/v2/xmetrics"
```

Comment:
@maurafortino do we need to use "github.com/xmidt-org/webpa-common/v2/xmetrics"?

Reply:
If `ancla` is already using `webpa-common` for metrics, this is fine for now.  We should move to `touchstone` ASAP, though.

If `ancla` doesn't have metrics, then we should use `touchstone` here.